Web/Scotty/Internal/Types.hs

```diff
@@ -47,7 +47,7 @@ import UnliftIO.Exception (Handler(..), catch, catches)
 
 
 --------------------- Options -----------------------
-data Options = Options { verbose :: Int -- ^ 0 = silent, 1(def) = startup banner
+data Options = Options { verbose :: Int -- ^ Verbosity level: 0 = silent, 1 (default) = startup text, print unhandled exceptions to stderr
```

Comment:
I wouldn't say _here_ that 1 is the default, as this is written in `defaultOptions`.